Packaging.props

```diff
@@ -19,6 +19,9 @@
     <!-- temporary change until buildtools can be updated https://github.com/dotnet/buildtools/issues/831 -->
     <LineupPackageVersion Condition="'$(LineupPackageVersion)' == ''">1.0.3</LineupPackageVersion>
     <PlatformPackageVersion Condition="'$(PlatformPackageVersion)' == ''">1.0.2</PlatformPackageVersion>
+
+    <!-- by default all packages will use the same version which revs with respect to product version -->
+    <Version Condition="'$(Version)' == ''">4.3.0</Version>
```

Comment:
I know you were already using this property but when I see this it worries me that we will conflict and it also isn't clear which version this is. Can we use PackageVersion instead?